analytics_data_api/v0/urls/learners.py

```diff
@@ -1,11 +1,17 @@
 from django.conf.urls import patterns, url
 
+from analytics_data_api.v0.urls import COURSE_ID_PATTERN
 from analytics_data_api.v0.views import learners as views
-from analytics_data_api.v0.urls import USERNAME_PATTERN
+
+USERNAME_PATTERN = r'(?P<username>.+)'
```

Comment:
I just noticed this, but is our `USERNAME_PATTERN` really `.+`? Seems pretty indiscriminate to me. [Here's the one we use in edx-platform](https://github.com/edx/edx-platform/blob/69c47e09511c0fda4762dc959a6a3025dc96f272/openedx/core/djangoapps/user_api/urls.py#L11). We don't have to address this in the current PR, but I wanted to make note regardless.